hexrd/fitting/calibration/calibrator.py

```diff
@@ -8,7 +8,6 @@ class Calibrator(ABC):
     @abstractmethod
     def type(self):
         """The type of the calibrator"""
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.